Makefile-libostree.am

```diff
@@ -149,6 +148,7 @@ endif
 
 if USE_LIBSOUP
 libostree_1_la_SOURCES += \
+    $(ENUM_GENERATED) \
```

Comment:
> Are you actually planning to use --with-soup=no? 

No, I discovered this issue only by accident, while trying to figure out why the build was failing for the issue fixed in the other patch.